cpmpy/solvers/pindakaas.py

```diff
@@ -0,0 +1,322 @@
+# /usr/bin/env python
+"""
+Interface to Pindakaas (`pkd`) API.
+
+Requires that the `pkd` library python package is installed:
+
+    $ pip install pindakaas
+
+`pkd` is a library to transform pseudo-Boolean and integer constraints into conjunctive normal form.
+See https://github.com/pindakaashq/pindakaas.
+
+This solver can be used if the model only has PB constraints.
+
+===============
+List of classes
+===============
+
+.. autosummary::
+    :nosignatures:
+
+    CPM_pindakaas
+
+==============
+Module details
+==============
+"""
+import inspect
+import time
+
+from ..exceptions import NotSupportedError
+from ..expressions.core import BoolVal, Comparison
+from ..expressions.variables import NegBoolView, _BoolVarImpl, _IntVarImpl
+from ..transformations.decompose_global import decompose_in_tree
+from ..transformations.flatten_model import flatten_constraint
+from ..transformations.get_variables import get_variables
+from ..transformations.linearize import linearize_constraint
+from ..transformations.normalize import simplify_boolean, toplevel_list
+from ..transformations.reification import only_bv_reifies, only_implies
+from .solver_interface import ExitStatus, SolverInterface
+
+
+class CPM_pindakaas(SolverInterface):
+    """
+    Interface to TEMPLATE's API.
```

Comment:
some TEMPLATE's not yet removed
(also, see the new way of starting a solver file and class doc)